### DISCUSSION TITLE
I can't deploy next app with middleware in my aws container

### DISCUSSION THREAD

**Discussion by jessemoe** [vercel/next.js#50431] [Help]
### Summary

I can't deploy next app with middleware in my aws container from circleCI .
The CI/CD deploy failed reason: 
`Too long with no output (exceeded 10m0s): context deadline exceeded`

### Additional information

```js
middleware.ts


// middleware.ts

import { NextRequest, NextResponse } from 'next/server'

const PUBLIC_FILE = /\.(.*)$/

export async function middleware(req: NextRequest) {
    if (
        req.nextUrl.pathname.startsWith('/_next') ||
        req.nextUrl.pathname.includes('/api/') ||
        PUBLIC_FILE.test(req.nextUrl.pathname)
    ) {
        return
    }

    if (req.nextUrl.locale === 'default') {
        const locale = req.cookies.get('NEXT_LOCALE')?.value || 'en'

        return NextResponse.redirect(
            new URL(`/${locale}${req.nextUrl.pathname}${req.nextUrl.search}`, req.url)
        )
    }
}

```
```


### Example

_No response_

**Comment by zmarkan** [comment]
Hi there, I work at CircleCI - this is due to there being no output in the terminal during a deployment job for too long and CircleCI basically thinks that the process has stopped responding and fails the job. 

This is probably not related to the middleware but to the commands used to deploy it.  I would check if there's a flag you could use to make sure that the deployment process generates more output while it's running, so CircleCI will know that the deploy is still going on.
Alternatively check the logs in AWS if there is anything out there that is silently failing.
